config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Comet - Short Time Historic",
+        "catalogue": "sth-comet",
+        "academy": "",
+        "readthedocs": "fiware-sth-comet",
+        "helpdesk": "STH-Comet",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-sth-comet"
+        ],
+        "docker": [
+            "fiware/sth-comet"
+        ]
+    },
+    {
+        "enabler": "Cosmos",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "fiware-cosmos-flink",
+        "helpdesk": "Cosmos",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-cosmos-orion-flink-connector",
+            "ging/fiware-cosmos-orion-flink-connector-examples",
+            "ging/fiware-cosmos-orion-spark-connector",
+            "ging/fiware-cosmos-orion-spark-connector-examples"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Cosmos - BigData Analysis",
+        "catalogue": "bigdata-analysis-cosmos",
+        "academy": "Cosmos",
+        "readthedocs": "fiware-cosmos",
+        "helpdesk": "Cosmos",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-cosmos",
+            "telefonicaid/fiware-sinfonier"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "CyberCAPTOR",
+        "catalogue": "cyber-security-ge-cybercaptor",
+        "academy": "Cyber seCurity",
+        "readthedocs": "cybercaptor",
+        "helpdesk": "Cyber Security",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": [
+            "fiware/cybercaptor-client",
+            "fiware/cybercaptor-cp2ds-peer",
+            "fiware/cybercaptor-p2ds-mgr",
+            "fiware/cybercaptor-server"
+        ]
+    },
+    {
+        "enabler": "Cygnus",
+        "catalogue": "cygnus",
+        "academy": "",
+        "readthedocs": "fiware-cygnus",
+        "helpdesk": "Cygnus",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-cygnus"
+        ],
+        "docker": [
+            "fiware/cygnus-ngsi"
+        ]
+    },
+    {
+        "enabler": "Docker",
+        "catalogue": "docker",
+        "academy": "Docker",
+        "readthedocs": "",
+        "helpdesk": "Docker",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": []
+    },
+    {
+        "enabler": "Domibus",
+        "catalogue": "electronic-data-exchange-domibus",
+        "academy": "Electronic Data Exchange",
+        "readthedocs": "",
+        "helpdesk": "Domibus",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
```

Comment:
"github": "fiware/context.Domibus",    ?

Reply:
There are no such repo and catalog points to ec.europa.eu

Reply:
In that case, it is better to leave it blank, because ec.europe.eu is not a github repository. Need to be checked by Jason if it is mandatory that there is a github repository (IMHO it was a requirements for passing from incubated GE to Develop (full) GE in the past.